sentences/de/_common.yaml

```diff
@@ -288,6 +310,20 @@ expansion_rules:
   sind_wurden: "(sind|wurden)"
   position: "{position} [Prozent|%]"
   volume: "{volume_level} [Prozent|%]"
+
+  # Timers
+  timer_set: "(starte|setze|stelle|erstelle)"
+  timer_cancel: "(beende|stoppe)"
+  timer_duration_seconds: "{timer_seconds:seconds} Sekunde[n]"
+  timer_duration_minutes: "{timer_minutes:minutes} Minute[n][ [und ]{timer_seconds:seconds} Sekunde[n]]"
+  timer_duration_hours: "{timer_hours:hours} Stunde[n][ [und ]{timer_minutes:minutes} Minute[n]][ [und ]{timer_seconds:seconds} Sekunde[n]]"
+  timer_duration: "<timer_duration_seconds>|<timer_duration_minutes>|<timer_duration_hours>"
+
+  timer_start_seconds: "{timer_seconds:start_seconds} Sekunde[n]"
+  timer_start_minutes: "{timer_minutes:start_minutes} Minute[n][ [und ]{timer_seconds:start_seconds} Sekunde[n]]"
+  timer_start_hours: "{timer_hours:start_hours} Stunde[n][ [und ]{timer_minutes:start_minutes} Minute[n]][ [und ]{timer_seconds:start_seconds} Sekunde[n]]"
```

Comment:
```suggestion
  timer_start_minutes: "{timer_minutes:start_minutes} Minute[n] [[und]{timer_seconds:start_seconds} Sekunde[n]]"
  timer_start_hours: "{timer_hours:start_hours} Stunde[n] [[und]{timer_minutes:start_minutes} Minute[n]] [[und]{timer_seconds:start_seconds} Sekunde[n]]"
```

Reply:
you forgot the space in [und ]. `12 Minuten und 5 Sekunden` would not match but `12 Minuten und5 Sekunden` would. I'll modify to keep tests successful.